plugins/kubernetes/app/models/kubernetes/resource.rb

```diff
@@ -115,6 +115,8 @@ def kind
       def desired_pod_count
         if @delete_resource
           0
+        elsif kind == "PodTemplate"
```

Comment:
not a fan ... all the other overrides work with inheritance

Reply:
right you are sir, fixed 👍